paasta_tools/setup_tron_namespace.py

```diff
@@ -162,6 +170,7 @@ def main():
     k8s_enabled_for_cluster = (
         yaml.safe_load(master_config).get("k8s_options", {}).get("enabled", False)
     )
+    new_configs: Dict[str, str] = {}  # service -> new_config
```

Comment:
til `create_complete_config` gives us back just a str; definitely not what i woulda expected by the name

Reply:
yea :(